.mention-bot

```diff
@@ -0,0 +1,4 @@
+{
+  "requiredOrgs": ["theforeman"]
+  "userBlackList": ["abenari", "unorthodoxgeek", "alongoldboim", "bgupta", "logicminds"]
```

Comment:
I think we should, at least from the committer group, but only @domcleal and @ohadlevy have the permissions to do that.